backend/src/StamAcasa.Common/Queue/NotificationQueueRabbit.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading.Tasks;
+using RabbitMQ.Client;
+using StamAcasa.Common.Notifications;
+using StamAcasa.Common.Services;
+
+namespace StamAcasa.Common.Queue
+{
+    public class NotificationQueueRabbit : IQueue
+    {
+        private string HostName { get; }
+
+        private string QueueName { get; }
+
+        private string Exchange { get; }
+
+        private string RoutingKey { get; }
+
+        public NotificationQueueRabbit(string hostName, string queueName, string exchange, string routingKey)
+        {
+            HostName = hostName;
+            QueueName = queueName;
+            Exchange = exchange;
+            RoutingKey = routingKey;
+        }
+
+        public Task<INotification> Dequeue()
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task Queue(INotification notification)
+        {
+            var factory = new ConnectionFactory() { HostName = HostName };
+            using (var connection = factory.CreateConnection())
```

Comment:
Aici am putea folosi un Pool Object pentru conexiunile la Queue. Si sa nu recreem de fiecare data conexiunea, o reutilizam.

Reply:
I made the requested changes but the PR seems to get stuck in building the Docker image steps ...